.github/actions/build-node/action.yml

```diff
@@ -1,50 +0,0 @@
-name: Agoric SDK build Node.js
-description: 'Build Node.js and cache built files'
-
-inputs:
-  node-version:
-    description: 'The version of Node.js to use'
-    required: true
-  cache-built:
-    description: 'Cache the result of the build'
-    type: boolean
-    required: false
-    default: true
-
-runs:
-  using: composite
-  steps:
-    - uses: actions/setup-node@v1
-      with:
-        node-version: ${{ inputs.node-version }}
-    - name: cache node modules
-      uses: actions/cache@v1
-      with:
-        path: ~/.cache/yarn
-        key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
```

Comment:
Would we not benefit from integrating the `~/.cache/yarn` cache into `restore-node` to speed up the first build of the revision?